src/class-elements/static-as-valid-instance-field.case

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2020 Igalia S.L. All rights reserved.
```

Comment:
what is a .case file for?

Reply:
It will create a version of this case for `ClassExpression` and another for `ClassDeclaration`.